### ISSUE TITLE
About the use of twitter dataset and loss function.

### ISSUE THREAD

**Issue by KLOSYX** [xiaolan98/BDANN-IJCNN2020#9]
Hi author! Your works are fantastic. I check your codes and data in the repo, and have a few questions as below:
- Is mediaeval 2015 or mediaeval 2016 do you use in your experiment? Or just a mixture of them? Seems that you only quote mediaeval 2015 in your paper, but I found images from 2016 test set in your data.
- You wrote `loss = class_loss - domain_loss` in your codes, but I think it should be `loss = class_loss + domain_loss`, is this normal? Why?

**Comment by xiaolan98** [comment]
Hi, I use mediaeval 2016 in our experiments. 

The loss should be loss = class_loss - domain_loss. It's the same as equation 10 in the paper.  We minimize the -domain_loss to make the model's prediction more robust to different events, especially the unseen event in the test set.

**Comment by KLOSYX** [comment]
> Hi, I use mediaeval 2016 in our experiments.
> 
> The loss should be loss = class_loss - domain_loss. It's the same as equation 10 in the paper. We minimize the -domain_loss to make the model's prediction more robust to different events, especially the unseen event in the test set.

Got it. Thank you for your reply. It's very helpful for my experiments.
